### PR TITLE
Replace cnv with zero

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

### CODE REVIEW

File: docs/source/deploy-model.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ In this way [ArgoCD][2] will be used to maintain your application always in sync
 
 Note: [AICoE Pipeline][3] can also update automatically the [imagestreamtag](https://github.com/thoth-station/elyra-aidevsecops-tutorial/blob/bb6fad2441e8df8aa56c2c0e6b5ac45a2cda42eb/manifests/overlays/test/imagestreamtag.yaml#L10) once a new release is created.
 
-Once everything is synced to the cluster, you can monitor your application from the [ArgoCD][2] using this [link](https://argocd-server-argocd.apps.cnv.massopen.cloud/applications) as shown in the image below:
+Once everything is synced to the cluster, you can monitor your application from the [ArgoCD][2] using this [link](https://argocd-server-argocd.apps.zero.massopen.cloud/applications) as shown in the image below:
```
Comment (goern) [inline docs/source/deploy-model.md:42]: I think the ArgoCD service migrated too.. @harshad16

File: docs/source/pre-requisite.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ You also need to have `OBJECT_STORAGE_ENDPOINT_URL`, `OBJECT_STORAGE_BUCKET_NAME
 
 ### Operate First KFP endpoint
 
-For [Operate First][1] you can find the Kubeflow Pipeline UI at this [link](http://ml-pipeline-ui-kubeflow.apps.cnv.massopen.cloud/#/pipelines).
+For [Operate First][1] you can find the Kubeflow Pipeline UI at this [link](http://ml-pipeline-ui-kubeflow.apps.zero.massopen.cloud/#/pipelines).
```
Comment (goern) [inline docs/source/pre-requisite.md:56]: this service is not available

File: docs/source/run-ai-pipeline.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ You can check the status of each step in the pipeline directly from the UI and d
 The model is stored in your bucket, you can check from your terminal using [aws CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html):
 
 ```bash
-  aws s3 --profile moc-pipeline-kfp --endpoint https://rgw-openshift-storage.apps.cnv.massopen.cloud/ ls s3://{your_bucket}/{your_project_name}/models/
+  aws s3 --profile moc-pipeline-kfp --endpoint https://rgw-openshift-storage.apps.zero.massopen.cloud/ ls s3://{your_bucket}/{your_project_name}/models/
```
Comment (goern) [inline docs/source/run-ai-pipeline.md:34]: this service is not available

File: src/model.py [hunk 4]
```diff
@@ -49,7 +49,7 @@ def __init__(self):
 
             s3_endpoint_url = os.getenv(
                 "ENDPOINT_URL",
-                "https://rgw-openshift-storage.apps.cnv.massopen.cloud/",
+                "https://rgw-openshift-storage.apps.zero.massopen.cloud/",
```
Comment (goern) [inline src/model.py:52]: not available

File: docs/source/pre-requisite.md (hunk 2 above)
Comment (pacospace) [reply docs/source/pre-requisite.md:56]: migration is not finished yet, I see all apps https://odh-dashboard-opf-dashboard.apps.zero.massopen.cloud/ not installed

File: docs/source/deploy-model.md (hunk 1 above)
Comment (tumido) [reply docs/source/deploy-model.md:42]: Actually, the ArgoCD service is located on the moc-infra cluster. Please change this url to:

https://argocd-server-argocd.apps.moc-infra.massopen.cloud/applications

File: src/model.py (hunk 4 above)
Comment (tumido) [reply src/model.py:52]: The proper endpoint here should be:

https://s3-openshift-storage.apps.zero.massopen.cloud/

**Review by tumido — CHANGES_REQUESTED** [review]

File: docs/source/deploy-model.md (hunk 1 above)
Comment (harshad16) [reply docs/source/deploy-model.md:42]: yes, correct url is the one @tumido  provided: https://argocd-server-argocd.apps.moc-infra.massopen.cloud/applications

File: docs/source/pre-requisite.md (hunk 2 above)
Comment (pacospace) [reply docs/source/pre-requisite.md:56]: @4n4nd what is the new endpoint for Kubeflow pipelines? Thanks!

Comment (tumido) [reply docs/source/pre-requisite.md:56]: @pacospace The ODH dashboard is lying for some reason. We need to fix that. (https://github.com/operate-first/odh-dashboard/issues/9) The components are available.

I think you can use this url:
http://istio-ingressgateway-istio-system.apps.zero.massopen.cloud/_/pipeline/#/pipelines

**Review by tumido — COMMENTED** [review]
/lgtm

File: docs/source/pre-requisite.md (hunk 2 above)
Comment (pacospace) [reply docs/source/pre-requisite.md]: Thanks @tumido, updated it!!

**Review by harshad16 — COMMENTED** [review]
/lgtm
thanks for the update

**Review by tumido — COMMENTED** [review]
/lgtm

**Review by tumido — COMMENTED** [review]
/lgtm

**Pull request merged**